ScoutSuite/providers/aws/resources/directconnect/base.py

```diff
@@ -6,9 +6,10 @@
 class Connections(AWSResources):
     async def fetch_all(self, **kwargs):
         raw_connections = await self.facade.directconnect.get_connections(self.scope['region'])
-        for raw_connection in raw_connections:
-            name, resource = self._parse_function(raw_connection)
-            self[name] = resource
+        if raw_connections:
```

Comment:
I think this is useless since `get_connections` never returns `None`. If `get_connections` fails, it returns an empty list and thus, iterating over it works as intented. What's the point of adding a condition like that, that adds a new level of indentation and introduce a git diff of 7 lines?